src/vlogger/vlogger.h

```diff
@@ -284,6 +284,14 @@
     logger("DOCA error: %s, %s. " log_fmt, doca_error_get_name(err), doca_error_get_descr(err),    \
            ##log_args)
 
+#define PRINT_DOCA_LEVEL(logger, level, err, log_fmt, log_args...)                                 \
+    logger(level, "DOCA error: %s, %s. " log_fmt, doca_error_get_name(err),                        \
+           doca_error_get_descr(err), ##log_args)
+
+#define PRINT_DOCA_ERR_RET(logger, err, log_fmt, log_args...)                                      \
```

Comment:
I prefer not to use return statetment inside macro. Its hard to read the code that use the macro because the return is hidden.

Reply:
Done.